components/map/basic/src/leaflet/layer-manager.js

```diff
@@ -0,0 +1,67 @@
+import L from 'leaflet'
+import { mapViewModes, mapViewNames } from './constants'
+
+export default class LayerManager {
+  constructor () {
+    this._currentLayer = 'normal.day'
+    this.layers = {}
+  }
+
+  createMapLayers ({mapViewModes, maxZoom, minZoom, appId, appCode}) {
+    let tileLayers = []
+    mapViewModes.forEach((value, index) => {
+      const baseMapView = this.getBaseMapView(value)
+      const tileLayer = L.tileLayer('https://{s}.{base}.maps.api.here.com/maptile/2.1/maptile/{mapID}/' + value + '/{z}/{x}/{y}/256/png8?app_id={app_id}&app_code={app_code}', {
+        app_code: appCode,
+        app_id: appId,
+        attribution: 'Map &copy; 1987-2017 <a href="http://developer.here.com">HERE</a>',
+        base: baseMapView,
+        mapID: 'newest',
```

Comment:
hardcoded ID ?

Reply:
Is not exactly and ID, i've changed the naming to have a more clear view about what it is. This is the 'version' of the tile. Newest will get the newset 'topographic' version of the tile